CMakeLists.txt

```diff
@@ -103,6 +103,13 @@ list(APPEND CMAKE_PREFIX_PATH /opt/rocm /opt/rocm/llvm /opt/rocm/hip /opt/rocm/h
 
 option(ENABLE_HIP_WORKAROUNDS Off)
 
+option(MIOPEN_USE_COMGR "When possible, use comgr to build kernels instead of offline tools" Off)
```

Comment:
Can you add `-DMIOPEN_USE_COMGR=On` to jenkins so it will be tested by the CI?

Reply:
We can set a new quick test to run through this with comgr compilation enabled

Reply:
@daniellowell Very good idea. Could you please? This can be done right away, independently of this PR.

Reply:
Sure.

Reply:
@daniellowell ...or perhaps it is better to add this quick test directly into this branch?